ncm-ccm/src/main/perl/ccm.pm

```diff
@@ -52,7 +52,11 @@ sub Configure
     delete($t->{version});
 
     while (my ($k, $v) = each(%$t)) {
-        print $fh "$k $v\n";
+        if (ref($v) eq 'ARRAY') {
```

Comment:
this will print warnings with scalars. `ref($v)` of scalar `$v` is undef, and `undef eq "string"` makes perl unhappy.
i would replace this with
```perl
my $value = ($ref($v) && $ref($v) eq 'ARRAY') ? join(',', @$v) : $v;
print $fh "$k $value\n";
```